sklego/ensemble_model.py

```diff
@@ -0,0 +1,87 @@
+from sklearn.linear_model import LogisticRegression
+from sklearn.metrics import confusion_matrix
+
+class EnsembleModel():
+    """
+    A classifier that takes the uncertainty p(class | model) of a classifier into account.
+    Based on the requirements of issue #23 of koaning/scikit-lego
+    """
+
+    def __init__(self, X_test, y_test, estimator=LogisticRegression(), alpha=0.5):
+        """
+
+        :param X_test: array of feature values that will be used to test the classifier and create a confusion matrix
+        :param y_test: array of target values that will be used to test the classifier and create a confusion matrix
+        :param estimator: sklean classifier;
+                          default one is LogisticRegression, but others can be used too (eg. Decision Tree)
+        :param alpha: hyperparameter; needs to be optimized to increase classification accuracy
+        """
+        self.estimator = estimator
+        self.alpha = alpha
+        self.X_test = X_test
+        self.y_test = y_test
+
+    def fit(self, X, y):
+        """
+
+        :param X: array of feature values used for training the classifier
+        :param y: array of target values used for training the classifier
+        :return: trained classifier (estimator)
+        """
+        self.fitted_estimator_ = self.estimator.fit(X, y)
+
+        return self
+
+    def predict(self, X):
+        """
+
+        :param X: array of feature values upon which a prediction is to be made
+        :return: prediction of the target value made by the trained classfifier
+        """
+        return self.fitted_estimator_.predict(X)
+
+    def predict_proba(self, X):
+        """
+
+        :param X: array of feature values upon which a prediction is to be made
```

Comment:
You probably want to check if the estimator is fitted. You can use `check_is_fitted` from `sklearn.utils.validation` like this `check_is_fitted(self, 'estimator_')`